signingscript/src/signingscript/task.py

```diff
@@ -53,6 +54,7 @@
         "autograph_authenticode_sha2": sign_authenticode_zip,
         "autograph_authenticode_sha2_stub": sign_authenticode_zip,
         "autograph_authenticode_sha2_rfc3161_stub": sign_authenticode_zip,
+        "autograph_authenticode_file": sign_authenticode_file,
```

Comment:
I doubt we can re-use sign_authenticode_file as-is.  It is currently set to use `context.config["authenticode_ca"]`, which is not the same intermediate that the EV cert chains to, and one of the `authenticode_cert*` config options as signing cert, which also point at the non-EV cert.
I'm seeing 2 options from a quick look:
- a new sign_authenticode_ev function that passes in the right EE and intermediate certs to winsign directly
- a new "keyid" for sign_authenticode_file, plus making it possible to pass in a different intermediate cert in e.g. `authenticode_ca_{keyid}` instead of always `authenticode_ca`
We'll also want to add both certs to signingscript/src/signingscript/data.

Reply:
Thanks, appreciate the pointer! I was pretty sure this was going to fail, but wasn't sure what I needed to do. This will help save me a lot of time investigating :)